Readme.md

```diff
@@ -8,8 +8,6 @@
 
 # Setup
 ```zsh
-git clone git@github.com:backslash-f/ShockEmu.git
-cd ShockEmu
```

Comment:
Any particular reason for removing these two lines?

Reply:
Just a personal preference not to have any direct url links or references to file structures in documentation, they can (and eventually will) change. Hopefully if people have found the repo on github, they know how to clone it :P

Reply:
Makes sense.